src/themes/ec/variables/_date-block.scss

```diff
@@ -2,10 +2,10 @@
 @use '../index' as *;
 
 $date-block: (
-  background-color-cancelled: var(--cm-surface-neutral-lowest),
+  background-color-cancelled: var(--cm-surface-transparent-01),
   background-color-default: var(--cm-surface-inverted),
   background-color-ongoing: var(--cm-surface-inverted),
-  background-color-past: var(--cm-surface-neutral-lowest),
+  background-color-past: var(--cm-surface-transparent-01),
```

Comment:
background color should be greyish for past/cancelled
![image](https://github.com/user-attachments/assets/955a3da7-3cdf-4666-a9f7-db8acbbffd46)